Topics/Development_Process/clean_coding_styling.md

```diff
@@ -0,0 +1,37 @@
+# Clean coding: coding and comment styling!
+
+
+### Why care about coding and commenting style?
+
+Code isn't good just cause it works. As we become more experienced in software engineering we soon realize that there is so much to consider other than the fact that it works. In fact, functioning code is the bare minimum. For example, most of us have learned that we need to consider time and space efficiency. If we didn't we'd constantly have to wait for the results we want. This can pose a big problem if those results are dependent on other results that are also not time efficient. We also have to think about scalability. We don't want to have to rewrite code just to add one feature. However, what's so important about coding and commenting style? How does that affect users? If I understand it is that not good enough? In short, no.
```

Comment:
Using 'cause' here seems a bit informal. Maybe you can consider using 'because' instead